src/core_modules/capture-core/components/CardList/CardListItem.component.js

```diff
@@ -233,7 +233,10 @@ const CardListItemIndex = ({
                                 {
                                     item.tei && item.tei.lastUpdated &&
                                     <div className={classes.smallerLetters}>
-                                        { i18n.t('Last updated') } {item.tei && moment(item.tei.lastUpdated).fromNow()}
+                                        { i18n.t('Last updated') }
+                                        { item.tei && <Tooltip content={item.tei.lastUpdated.toLocaleString()}>
+                                            { moment(item.tei.lastUpdated).fromNow() }
```

Comment:
Let's add `fromServerDate` to properly handle the time difference between the client and the server.
When enrolling and the list of possible duplicates is displayed, `item.tei.lastUpdated` doesn't show me anything. Does it work for you? I could only see the date when I used `item.tei.updatedAt` instead.

Reply:
No, it doesn't show unless `updatedAt` is used. Will update the code